docs/content/tools/xdebug.md

```diff
@@ -138,31 +138,43 @@ You can run your scripts in console and debug them in the same way as browser re
           "cwd": "${fileDirname}",
           "port": 9000,
           "pathMappings": {
-            "/var/www/": "${workspaceRoot}"
+            "/var/www/": "${workspaceFolder}"
           }
         }
       ]
     }
     ```
-    
+
 4. Set a breakpoint
 5. Select the **Listen for XDebug** configuration from the dropdown and click **Start Debugging**
 
 You can debug both web requests and cli scripts using this configuration.
 
+### CLI Scripts: Drush {#vscode-drush}
+
+To debug Drush commands using Xdebug and VSCode add the following to your path mappings under the configuration that begins with `"name": "Listen for XDebug",`
```

Comment:
please add comma: ... and VSCode, add the following...

Reply:
done!